imagefiles/dockcross

```diff
@@ -180,12 +180,17 @@ fi
 # Now, finally, run the command in a container
 #
 tty -s && TTY_ARGS=-ti || TTY_ARGS=
-docker run $TTY_ARGS --rm \
+CONTAINER_NAME=dockcross
```

Comment:
`_dockcross` instead of `dockcross` could avoid conflicts if folks happen to use a dockcross image manually and name it `dockcross`.